docs/architecture/adr-044-protobuf-updates-guidelines.md

```diff
@@ -0,0 +1,103 @@
+# ADR 044: Guidelines for Updating Protobuf Definitions
+
+## Changelog
+
+- 28.06.2021: Initial Draft
+
+## Status
+
+Draft
+
+## Abstract
+
+This ADR provides guidelines and recommended practices when updating Protobuf definitions. These guidelines are targeted at module developers.
```

Comment:
this is a good example where it's halfway between ADR and docs.

Talking with some people, we decided to go with an ADR, as it may provide more visibility for other chain & client developers

Reply:
Definitely makes sense to me!